### PR TITLE
[don't merge] CI improvements for reporting changes in actual templates

### PR DESCRIPTION
**Pull request by piontec**
Visualize what the final change will be - checks comments at the bottom

parameters to `diff` still need tuning

### CODE REVIEW

File: bases/app_sets/hello-web-app/default_config_hello_world.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ db_config:
   db_port: 5433 # has to be in sync with config in default_config_simple_db.yaml
   db_user: hello-web-app # has to be in sync with config in default_config_simple_db.yaml
   host: simple-db.simple-web.svc # has to be in sync with 'namespace:' set in kustomization.yaml
-thread_pool_size: 16
+thread_pool_size: 17
```
Comment (uvegla) [inline bases/app_sets/hello-web-app/default_config_hello_world.yaml:6]: Do you want to keep this or was just for testing the diff?

File: tools/test-all-ffb [hunk 2]
```diff
@@ -7,12 +7,14 @@ help() {
...
 
+Usage: $0 [validate|template]
+
 "
     exit 1
 }
 
 test() {
-
+    mode="$1"
```
Comment (uvegla) [inline tools/test-all-ffb:17]: I think this is not used.

**Review by uvegla — APPROVED** [review]

**Review by kubasobon — APPROVED** [review]

File: bases/app_sets/hello-web-app/default_config_hello_world.yaml (hunk 1 above)
Comment (piontec) [reply bases/app_sets/hello-web-app/default_config_hello_world.yaml:6]: I will revert just before merging

File: tools/test-all-ffb (hunk 2 above)
Comment (piontec) [reply tools/test-all-ffb:17]: Yup, my bad - fixed.

**Comment by piontec** [comment]
> PR has `don't merge` in the title Gets merged :trollface:

Yeah, I removed all the reasons for 'don't merge', but forgot about the title :D

**Pull request merged**